scripts/release-pr-steps.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+# Exit on error and output commands.
+set -ex
+
+NEXT_VERSION=$1
+if [ "$NEXT_VERSION" = "" ]; then
+	echo "Error: Version not set!"
+	exit
+fi
+
+RELEASE_BRANCH=$2
+if [ "$RELEASE_BRANCH" = "" ]; then
+	echo "Error: Release branch not set!"
+	exit
+fi
+
+# Fix permissions for ssh config and keys.
+chown -R root:root /root/.ssh
+export GIT_SSH_COMMAND="ssh -i /root/.ssh/id_rsa"
+
+# Create working directory and checkout Sensei.
+mkdir /release && cd /release
+git clone git@github.com:Automattic/sensei.git
+cd /release/sensei
+
+# Checkout release branch.
+git switch $RELEASE_BRANCH
+
+# Exit if could not switch to release branch.
+if [ $? -ne 0 ]; then
+	echo "Error: Could not switch to release branch!"
+	exit
+fi
+
+# Install dependencies.
+composer install && npm ci
+
+# Disable commit signing.
+git config --global commit.gpgsign false
+
+# Create the release branch.
+git switch -c "release/$NEXT_VERSION"
+
+echo "Replace next version tag"
+./scripts/replace-next-version-tag.sh $NEXT_VERSION
+if [[ -n $(git status -s) ]]; then
+	git add .
+	git commit -m 'Replace next version tag'
+else
+  echo "There are no changes after next version tag replacement."
+fi
+
+echo "Update plugin version"
+update-version.sh $NEXT_VERSION
+if [[ -n $(git status -s) ]]; then
+	git add .
+	git commit -m 'Update plugin version'
+else
+  echo "There are no changes after updating the version in plugin files."
+fi
+
+echo "Changlogger write"
+# Write changelog.
+composer exec -- changelogger write
+# Copy changelog section in readme.
+./scripts/copy-changelog-to-readme.php
+if [[ -n $(git status -s) ]]; then
+	git add .
+	git commit -m 'Update chaneglog'
+else
+  echo "There are no changes after running changelogger."
+fi
+
+echo "Build translations"
+npm run build:assets && wp i18n make-pot --exclude=build,lib,vendor,node_modules,assets/vendor --headers='{"Last-Translator":null,"Language-Team":null,"Report-Msgid-Bugs-To":"https://wordpress.org/support/plugin/sensei-lms"}' . lang/sensei-lms.pot --allow-root
```

Comment:
What do you think to move the wp 18n command to the package json and make it as part of the build:assets?

Reply:
The script exists already. However, I had to add `--allow-root` flag into it to make it working in a Docker container.

Reply:
Would this work?

```
npm run i18n:build -- --allow-root
```

Reply:
Doesn't work that way, unfortunately.
Maybe it's better to use my original approach without using `npm run` shortcut?

Reply:
Yep, thank you, it's working.
The PR has been updated.